portworx/README.md

```diff
@@ -55,8 +64,48 @@ The following values must be defined, either through `helm install --set ...` or
 * etcdEndPoint     :   of the form `https://portal-ssl294-1.bmix-wdc-yp-a7a89461-abcc-45e5-84d7-cde68723e30d.588786498.composedb.com:15832`
                        where the actual URLs correspond to your `etcd` URL.
 
-```
-helm install --debug --name portworx-ds --no-hooks 
+>**Note:** For baremetal instances, please specify `dataInterface=bond0` and `managementInterface=bond0`
+
+>**Note:** For disks, the default is to use all unmounted/unformatted disks and partitions.  Disk resources can be added explicitly via `drives=/dev/sda;/dev/sdb;...` 
```

Comment:
I think we can also mention about multipath as in IBM Cloud we use multipath devices such as `/dev/dm-0` for example.